sdk_container/src/third_party/coreos-overlay/coreos-base/ue-rs/ue-rs-0.1.0-r1.ebuild

```diff
@@ -0,0 +1 @@
+ue-rs-9999.ebuild
```

Comment:
I'm not sure if having `ue-rs-9999` makes any sense. Could we have `ue-rs-0.1.0-r1.ebuild` as a regular file and drop the `9999` one?

Reply:
i would keep this as is

Reply:
`git blame` should be smart enough to follow renames. But I'm not going to block the PR on this.